cleanlab/filter.py

```diff
@@ -53,18 +54,18 @@
 
 
 def find_label_issues(
-    labels,
-    pred_probs,
+    labels: np.ndarray,
+    pred_probs: Optional[np.ndarray],
     *,
     return_indices_ranked_by=None,
-    rank_by_kwargs={},
+    rank_by_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
This is a bit more secure since, dict and list objects as default values are kinda "static".
I also don't really know what type the `return_indices_ranked_by` should be.

Reply:
Right now, those are strings.